ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -752,6 +748,12 @@ public Builder tableRefreshIntervalMs(final long tableRefreshIntervalMs) {
       return this;
     }
 
+    public Builder cleanPeerTableIntervalMs(final long cleanPeerTableIntervalMs) {
```

Comment:
This method appears to not have test coverage, all the other builder methods do.

Reply:
I just added the method for completeness of the builder, it isn't actually used.  Do you think we need a test for this?